irohad/ametsuchi/CMakeLists.txt

```diff
@@ -114,6 +114,9 @@ add_library(ametsuchi
     impl/in_memory_block_storage.cpp
     impl/in_memory_block_storage_factory.cpp
     )
+target_link_libraries(ametsuchi 
+    async_subscription
+    )
```

Comment:
```suggestion
```
Already linked in `application`, tests should not be linked to `async_subscription`.

Reply:
It fails to link without this.